packages/create-svelte/shared/+skeletonlib+checkjs/jsconfig.json

```diff
@@ -9,6 +9,6 @@
 		"skipLibCheck": true,
 		"sourceMap": true,
 		"strict": true,
-		"moduleResolution": "NodeNext"
+		"moduleResolution": "bundler"
```

Comment:
sorry for the confusion - these should stay as `NodeNext`. I meant to set `bundler` for the skeleton app and showcase app, not the lib